addon/src/lib/middleware/Hub.js

```diff
@@ -14,8 +14,8 @@ export default class Hub {
   dispatch: Dispatch;
   ports: Set<EventEmitter>;
   constructor() {
-    this.dispatch = () =>
-      console.error('Hub cannot use dispatch() before middleware()');
+    this.dispatch = () => // eslint-disable-next-line no-console
+    console.error('Hub cannot use dispatch() before middleware()');
```

Comment:
Offtopic: Should we do something more than `console.error()` here, like emit some error to Sentry or whatever?
Or do we care if errors get sent into a Browser Console abyss?